modules/reference/pages/rpk/rpk-connect/rpk-connect-help.adoc

```diff
@@ -0,0 +1,82 @@
+= rpk connect help
+
+A stream processor for mundane tasks.
+
+Either run Redpanda Connect as a stream processor or choose a command:
+
+Available commands:
+- xref:reference:rpk/rpk-connect/rpk-connect-echo.adoc[] - Parse a config file and echo back a normalized version.
+
+- xref:reference:rpk/rpk-connect/rpk-connect-lint.adoc[] - Parse Redpanda Connect configs and report any linting errors.
+
+- xref:reference:rpk/rpk-connect/rpk-connect-streams.adoc[] - Run Redpanda Connect in streams mode.
+
+- xref:reference:rpk/rpk-connect/rpk-connect-list.adoc[] - List all Redpanda Connect component types.
+
+- xref:reference:rpk/rpk-connect/rpk-connect-create.adoc[] - Create a new Redpanda Connect config.
+
+- xref:reference:rpk/rpk-connect/rpk-connect-test.adoc[] - Execute Redpanda Connect unit tests.
+
+- xref:reference:rpk/rpk-connect/rpk-connect-template.adoc[] - Interact and generate Redpanda Connect templates.
+
+- xref:reference:rpk/rpk-connect/rpk-connect-blobl.adoc[] - Execute a Redpanda Connect mapping on documents consumed via stdin.
+
+- xref:reference:rpk/rpk-connect/rpk-connect-studio.adoc[] - Interact with Benthos studio (https://studio.benthos.dev).
+
+- xref:reference:rpk/rpk-connect/rpk-connect-help.adoc[] - Shows a list of commands or help for one command.
+
+See https://docs.redpanda.com/redpanda-connect
```

Comment:
@JakeSCahill should we refer to another page? What's the specific syntax that we're going to use for these cross-links?